__test__/remove.test.js

```diff
@@ -0,0 +1,34 @@
+/* import { iteratee } from 'lodash'; */
+// import { get } from 'lodash';
```

Comment:
- [ ] As already suggested, please kindly remove all code comments as it pollutes the codebase